pypco/pco.py

```diff
@@ -425,10 +429,10 @@ def iterate(self, url, offset=0, per_page=25, **params): #pylint: disable=too-ma
                 }
 
                 if 'can_include' in response['meta']:
-                    record['meta']['can_include']: response['meta']['can_include']
```

Comment:
this is invalid python code (since : is not allowed here or in walrus orator assuming typo and changing it to =)

this got discovered by mypy

Reply:
Definitely a typo, interesting that the interpreter doesn't complain about it. I'd like to get a unit tests on this to make sure we're actually updating 'can_include' before we merge this in to master.